platform/packages/finance/src/price/mod.rs

```diff
@@ -174,9 +175,14 @@ where
     fn lossy_add(self, rhs: Self) -> Option<Self> {
         const FACTOR: Amount = 1_000_000_000_000_000_000; // 1*10^18
         let factored_amount = FACTOR.into();
```

Comment:
the `fn Coin<C>::new()`  is const so please switch to using `Coin<C>` for `FACTOR`